libraries/honeycomb.rb

```diff
@@ -1,7 +1,8 @@
 require "chef/http/simple_json"
 require "time"
+require "securerandom" unless defined?(SecureRandom)
 
-VERSION="0.1.1"
+VERSION="1.0.0"
```

Comment:
We'll create a subsequent release PR to bump the version.

```suggestion
VERSION="0.1.1"
```